specs-jetpack-calypso/wp-jetpack-plans-spec.js

```diff
@@ -90,4 +94,28 @@ describe( `[${ host }] Jetpack Plans: (${ screenSize }) @jetpack`, function() {
 			await shoppingCartWidgetComponent.empty();
 		} );
 	} );
+
+	describe( 'Renew Premium Plan:', function() {
+		before( async function() {
+			return await driverManager.clearCookiesAndDeleteLocalStorage( driver );
+		} );
+
+		step( 'Can log into WordPress.com', async function() {
+			this.loginFlow = new LoginFlow( driver, 'jetpackUserPREMIUM' );
+			return await this.loginFlow.login();
+		} );
+
+		step( 'Can renew Premium plan', async function() {
+			const navBarComponent = await NavBarComponent.Expect( driver );
+			await navBarComponent.clickProfileLink();
+			const profilePage = await ProfilePage.Expect( driver );
+			await profilePage.chooseManagePurchases();
+			const purchasesPage = await PurchasesPage.Expect( driver );
+			await purchasesPage.dismissGuidedTour();
+			await purchasesPage.selectPremiumPlan();
```

Comment:
I should probably use a different way of selecting the relevant plan: Right now, there's the `jetpackbleedingedgepremium` disconnected site with a Premium plan at the top of `/me/purchases/. https://github.com/Automattic/wp-calypso/pull/25743 adds a 'Manage Purchase' link to it, so the e2e test will accidentally pick that -- but that purchase doesn't even have a 'Renew Plan' link, so the test fails for the wrong reason (and will continue to do so, even if a working version of https://github.com/Automattic/wp-calypso/pull/25743 is merged into Calypso).

Reply:
I decided to add an `is-disconnected-site` class on the Calypso side https://github.com/Automattic/wp-calypso/pull/26055 which I think should make the selector more idiomatic, and stabler...